src/main/java/frc/robot/lib/states/Claw.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.lib.states;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import frc.robot.RobotMap;
+
+/** Add your docs here. */
+public class Claw {
+
+    // TODO: positions
+    double closeCubePosition = 0.0;
+    double closeConePosition = 0.0;
+
+    public Claw() {
+        RobotMap.claw.setNeutralMode(NeutralMode.Brake);
```

Comment:
The claw variable needs to be created in RobotMap. It should also be more descriptive that just "claw". It is a motor class that commands the motor to open and close the claw. The name should reflect that.